build/contracts/StandardTokenData.json

```diff
@@ -0,0 +1,5419 @@
+{
```

Comment:
Is this needed? It can be imported like this:
`import StandardTokenData from '@gnosis.pm/util-contracts/build/contracts/StandardTokenData.json'`